internal/boxcli/global.go

```diff
@@ -85,7 +105,23 @@ func pullGlobalCmdFunc(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return errors.WithStack(err)
 	}
-	return box.PullGlobal(cmd.Context(), args[0])
+	err = box.PullGlobal(cmd.Context(), overwrite, args[0])
+	if errors.Is(err, pullbox.ErrFileExists) {
```

Comment:
This breaks the Devbox interface abstraction by having callers check for an internal package (`pullbox`) error kind. This works for internal callers but some external user of the devbox package won't be able to use it.

Could we re-use `fs.ErrExist` instead of defining a new Err kind?